### PR TITLE
Use merge table in the dashboard; add Cloud overview to the system table

### PR DESCRIPTION
**Pull request by alexey-milovidov**
### Changelog category (leave one):
- Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
The dashboard will display metrics across different ClickHouse versions even if the structure of system tables has changed after the upgrade.

### CODE REVIEW

**Comment by robot-clickhouse-ci-1** [comment]
*This is an automated comment for commit 27f115f581021e6b3367a6328abb65d8c22ee21b with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/59967/27f115f581021e6b3367a6328abb65d8c22ee21b/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/59967/27f115f581021e6b3367a6328abb65d8c22ee21b/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

**Comment by alexey-milovidov** [comment]
Note: when resolving the conflict with the private repository, also change "overview" to "Cloud overview" in the `dashboard.html`.

**Comment by nikitamikhaylov** [comment]
Can you please also add a dashboard with ProfileEvent_MergedRows?

File: src/Storages/System/StorageSystemDashboards.cpp [hunk 1]
```diff
@@ -26,192 +26,329 @@ void StorageSystemDashboards::fillData(MutableColumns & res_columns, ContextPtr,
...
 WHERE event_date >= toDate(now() - {seconds:UInt32}) AND event_time >= now() - {seconds:UInt32} AND metric = 'MaxPartCountForPartition'
 GROUP BY t
 ORDER BY t WITH FILL STEP {rounding:UInt32}
 )EOQ") }
+        },
+        /// Default dashboard for ClickHouse Cloud
+        {
+            { "dashboard", "Cloud overview" },
+            { "title", "Queries/second" },
+            { "query", "SELECT \n  toStartOfInterval(event_time, INTERVAL {rounding:UInt32} SECOND)::INT AS t,\n  avg(metric)\nFROM (\n  SELECT event_time, sum(ProfileEvent_Query) AS metric \n  FROM clusterAllReplicas(default, merge('system', '^metric_log'))\n  WHERE event_date >= toDate(now() - {seconds:UInt32})\n    AND event_time >= now() - {seconds:UInt32}\n  GROUP BY event_time)\nGROUP BY t\nORDER BY t WITH FILL STEP {rounding:UInt32} SETTINGS skip_unavailable_shards = 1" }
```
Comment (antaljanosbenjamin) [inline src/Storages/System/StorageSystemDashboards.cpp:221]: Can we format the queries similary to above? It makes readability much better.

**Review by antaljanosbenjamin — CHANGES_REQUESTED** [review]

**Review by antaljanosbenjamin — APPROVED** [review]

**Pull request merged**